config/src/client.rs

```diff
@@ -106,3 +106,75 @@ impl Default for ConfigurationProxy {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use iroha_config_base::proxy::{Builder, LoadFromDisk};
+    use proptest::prelude::*;
+
+    use super::*;
+    use crate::torii::{uri::DEFAULT_API_URL, DEFAULT_TORII_TELEMETRY_URL};
+
+    const CONFIGURATION_PATH: &str = "../configs/client_cli/config.json";
+
+    #[allow(clippy::expect_used)]
+    fn placeholder_keypair() -> KeyPair {
+        let public_key = "ed01207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
+            .parse()
+            .expect("Public key not in mulithash format");
+        let private_key = PrivateKey::from_hex(
+            Algorithm::Ed25519,
+            "9ac47abf59b356e0bd7dcbbbb4dec080e302156a48ca907e47cb6aea1d32719e7233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
+        ).expect("Private key not hex encoded");
+
+        KeyPair::new(public_key, private_key).expect("Key pair mismatch")
+    }
+
+    #[allow(clippy::expect_used)]
+    fn placeholder_account() -> <Account as Identifiable>::Id {
+        AccountId::from_str("alice@wonderland").expect("Account ID not valid")
+    }
+
+    fn arb_keys() -> BoxedStrategy<(Option<PublicKey>, Option<PrivateKey>)> {
```

Comment:
I'm not 100% sure about this. Most keys are completely random so a proptest does literally nothing. Could we run tests with edge cases (one missing key, other missing key, both present but don't match), and leave this out of the proptest?